odbc/src/api/definitions.rs

```diff
@@ -1,739 +1,2 @@
-#[allow(unused)]
-pub const SQL_FN_STR_LCASE: u32 = 0x00000040;
-#[allow(unused)]
-pub const SQL_FN_STR_REPEAT: u32 = 0x00000080;
-#[allow(unused)]
-pub const SQL_FN_STR_REPLACE: u32 = 0x00000100;
-#[allow(unused)]
-pub const SQL_FN_STR_SUBSTRING: u32 = 0x00000800;
-#[allow(unused)]
-pub const SQL_FN_STR_UCASE: u32 = 0x00001000;
-#[allow(unused)]
-pub const SQL_FN_STR_ASCII: u32 = 0x00002000;
-#[allow(unused)]
-pub const SQL_FN_STR_CHAR: u32 = 0x00004000;
-#[allow(unused)]
-pub const SQL_FN_STR_DIFFERENCE: u32 = 0x00008000;
-#[allow(unused)]
-pub const SQL_FN_STR_LOCATE_2: u32 = 0x00010000;
-#[allow(unused)]
-pub const SQL_FN_STR_SOUNDEX: u32 = 0x00020000;
-#[allow(unused)]
-pub const SQL_FN_STR_SPACE: u32 = 0x00040000;
-#[allow(unused)]
-pub const SQL_FN_STR_BIT_LENGTH: u32 = 0x00080000;
-#[allow(unused)]
-pub const SQL_FN_STR_CHAR_LENGTH: u32 = 0x00100000;
-#[allow(unused)]
-pub const SQL_FN_STR_CHARACTER_LENGTH: u32 = 0x00200000;
-#[allow(unused)]
-pub const SQL_FN_STR_OCTET_LENGTH: u32 = 0x00400000;
-#[allow(unused)]
-pub const SQL_FN_STR_POSITION: u32 = 0x00800000;
-
-// SQL_TIMEDATE_FUNCTIONS functions
-pub const SQL_FN_TD_CURRENT_TIMESTAMP: u32 = 0x00080000;
-pub const SQL_FN_TD_EXTRACT: u32 = 0x00100000;
-
-// SQL_CATALOG_USAGE bitmasks
-pub const SQL_CU_DML_STATEMENTS: u32 = 0x00000001;
-
-// SQL_GETDATA_EXTENSIONS bitmasks
-pub const SQL_GD_ANY_COLUMN: u32 = 0x00000001;
-pub const SQL_GD_ANY_ORDER: u32 = 0x00000002;
-
-// SQL_TIMEDATE_ADD_INTERVALS and SQL_TIMEDATE_DIFF_INTERVALS functions
-pub const SQL_FN_TSI_SECOND: u32 = 0x00000002;
-pub const SQL_FN_TSI_MINUTE: u32 = 0x00000004;
-pub const SQL_FN_TSI_HOUR: u32 = 0x00000008;
-pub const SQL_FN_TSI_DAY: u32 = 0x00000010;
-pub const SQL_FN_TSI_WEEK: u32 = 0x00000020;
-pub const SQL_FN_TSI_MONTH: u32 = 0x00000040;
-pub const SQL_FN_TSI_QUARTER: u32 = 0x00000080;
-pub const SQL_FN_TSI_YEAR: u32 = 0x00000100;
-
-// SQL_SQL92_PREDICATES bitmasks
-pub const SQL_SP_EXISTS: u32 = 0x00000001;
-pub const SQL_SP_ISNOTNULL: u32 = 0x00000002;
-pub const SQL_SP_ISNULL: u32 = 0x00000004;
-pub const SQL_SP_LIKE: u32 = 0x00000200;
-pub const SQL_SP_IN: u32 = 0x00000400;
-pub const SQL_SP_BETWEEN: u32 = 0x00000800;
-pub const SQL_SP_COMPARISON: u32 = 0x00001000;
-pub const SQL_SP_QUANTIFIED_COMPARISON: u32 = 0x00002000;
-
-// SQL_SQL92_RELATIONAL_JOIN_OPERATORS bitmasks
-pub const SQL_SRJO_CROSS_JOIN: u32 = 0x00000002;
-pub const SQL_SRJO_INNER_JOIN: u32 = 0x00000010;
-pub const SQL_SRJO_LEFT_OUTER_JOIN: u32 = 0x00000040;
-pub const SQL_SRJO_RIGHT_OUTER_JOIN: u32 = 0x00000100;
-
-// SQL_AGGREGATE_FUNCTIONS bitmasks
-pub const SQL_AF_AVG: u32 = 0x00000001;
-pub const SQL_AF_COUNT: u32 = 0x00000002;
-pub const SQL_AF_MAX: u32 = 0x00000004;
-pub const SQL_AF_MIN: u32 = 0x00000008;
-pub const SQL_AF_SUM: u32 = 0x00000010;
-pub const SQL_AF_DISTINCT: u32 = 0x00000020;
-pub const SQL_AF_ALL: u32 = 0x00000040;
+// SQL-1817: Move all definitions to odbc-sys
+pub use mongo_odbc_core::api_definitions::*;
```

Comment:
I don't think this was necessary for this PR, but I can see where you are coming from though, because you wanted to use odbc_version you now needed more things from definition.
Having the ODBC api definitions in core bothers me more than I thought. 
What about a `definition` package similar to the `constant` package that both core and  odbc can use?

Reply:
That would work, but maybe let's think about what else we want to do organizationally with the driver and get all the work done at once, or at least planned out rather than doing it piecemeal.